frappe/geo/languages.csv

```diff
@@ -14,6 +14,7 @@ el,Ελληνικά,0
 en,English,0
 en-GB,English (United Kingdom),0
 en-US,English (United States),0
+eo,In-Context Translation,0

```

Comment:
esperanto code is by design or some hack to make it work?

Reply:
Some hack (just any unused language code)